lib/cts/cases/create.rb

```diff
@@ -84,6 +101,14 @@ def set_received_date
       end
     end
 
+    def set_external_deadline
+      if options.key? :external_deadline
+        options[:external_deadline]
+      else
+        0.business_days.after(4.business_days.since)
+      end
```

Comment:
don't understand this.  4 business days since what?

Reply:
it's since today just need 4 days in the future. can set it as a variable with a name to make it clearer?

Reply:
The call to `0.business_days.after` appears redundant here.

`4.business_days.since` returns a business day, and then `0.business_days.after` returns the current / next business day after date provided. If anything these calls should be the other way around. But, in tests we use `0.business_days.ago` to get around problems from running tests outside of business hours, and I'm not completely sure why we have `after` here.

I have a feeling that what we really want here, and in other places where we do the 4-business-day calculation, is:

```
  4.business_days.after(0.business_days.ago)
```

My guess is that this crept in when we (I?) didn't understand the business time stuff so well.